spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/subscriber/PubSubSubscriberTemplate.java

```diff
@@ -109,9 +109,9 @@ public void setMessageConverter(PubSubMessageConverter pubSubMessageConverter) {
 		this.pubSubMessageConverter = pubSubMessageConverter;
 	}
 
-	public void setAckExecutor(Executor ackExecutor) {
-		Assert.notNull(ackExecutor, "ackExecutor can't be null.");
-		this.ackExecutor = ackExecutor;
+	public void setCallbackExecutor(Executor callbackExecutor) {
```

Comment:
Do you realize that this is a breaking change? We just can't rename API just because.
Please, consider to deprecate exiting method in favor of a new one.
And let's keep it here at least for the next version!

Reply:
Point well taken. If this is the direction in which this code will go, I'll add the proper deprecation (I hope it does not go there, though -- per discussion below, I would prefer to revert the changes to `PubSubSubscriberTemplate` and manage multi threading in `PubSubReactiveFactory` instead).